src/app/test_executive/payments_test.ml

```diff
@@ -473,15 +473,17 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
     let%bind () =
       section_hard
         "change snark worker key from snark-node-key1 to snark-node-key2"
-        (let%bind () =
-           Integration_test_lib.Graphql_requests.must_set_snark_worker ~logger
-             (Network.Node.get_ingress_uri snark_coordinator)
-             ~new_snark_pub_key:
-               ( snark_node_key2.keypair.public_key
-               |> Signature_lib.Public_key.compress )
-         in
-         (* adding a wait for 3 minutes so that the new snark worker can start.  this is a temporary solution to what's essentially a race condition causing lots of flakiness/non-determinism for this test *)
-         Malleable_error.lift (Async.after (Time.Span.of_int_sec 180)) )
+        (Integration_test_lib.Graphql_requests.must_set_snark_worker ~logger
+           (Network.Node.get_ingress_uri snark_coordinator)
+           ~new_snark_pub_key:
+             ( snark_node_key2.keypair.public_key
+             |> Signature_lib.Public_key.compress ) )
+    in
+    let%bind () =
+      section_hard "change snark work fee from 0.0002 to 0.0001"
+        (Integration_test_lib.Graphql_requests.must_set_snark_work_fee ~logger
```

Comment:
nitpick: please introduce a variable called "new_snark_fee" or something like that, and use that variable instead of just directly copy pasting the new value in all the various places

Reply:
There is a variable called `key_2_balance_expected`. When we set the new snark fee, it's using nanomina instead of mina. I would try to combine those 2 if that's not too verbose to do the conversion.